ui/assets/css/_fonts.scss

```diff
@@ -0,0 +1,52 @@
+/*
+ Fonts
+ ==========================================================================
+ Place all @font-face rules in fonts.scss
+ */
+
+ @font-face {
+   font-family: 'Source Sans Pro';
+   font-style: normal;
+   font-weight: 300;
+   src: url('/font/SourceSansPro-Light.ttf');
```

Comment:
Ah yes, I forgot to swing back around and update these. Relative paths didn't seem to work but I updated to include the _right_ absolute path